bin/proxy.js

```diff
@@ -0,0 +1,45 @@
+require('babel-register');
+
+const http = require('http');
+
+const bunyan = require('bunyan');
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+const log = bunyan.createLogger({
+  name: 'proxy',
+  app: config.get('appName'),
+  serializers: bunyan.stdSerializers,
+});
+
+const proxy = httpProxy.createProxyServer();
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
+const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+
+log.info(`apiHost: ${apiHost}`);
+log.info(`frontendHost: ${frontendHost}`);
+
+function getHost(req) {
+  if (!req.headers['user-agent'].includes('Android') || req.url.startsWith('/api/')) {
```

Comment:
what were your thoughts on replacing the user agent check with a whitelist of `/developers`, `/editors`, and `/admin` (or whatever) ? That will prep us for inevitable desktop site development. Or will it be annoying not having a working index page for the desktop site?